source/containers.tex

```diff
@@ -21741,11 +21741,11 @@
 \begin{itemdescr}
 \pnum
 \constraints
-\tcode{sizeof...(slices)} equals \tcode{Extents::rank()}.
+\tcode{sizeof...(slices)} equals \tcode{decltype(src)::rank()}.
```

Comment:
`decltype(src) is `const extents<IndexType, Extents...>&` and reference types don't have members, so this is ill-formed. `src.rank()` would work here, as would `extents<IndexType, Extents...>::rank()`. (I think the latter is more conventional for the Standard Library, though I have no data to back that up nor strong feelings either way.)